package.json

```diff
@@ -25,7 +25,9 @@
   },
   "homepage": "https://github.com/parse-community/parse-server-s3-adapter#readme",
   "dependencies": {
-    "aws-sdk": "^2.59.0"
+    "aws-sdk": "^2.59.0",
+    "parse": "^2.8.0",
+    "parse-server": "^3.9.0"
```

Comment:
Remove the parse-server dependency?
Pin the parse dependency? (remove the ^) we should pin all dependencies as some point.

Reply:
Guess I don't understand why.   How should I deal with throwing Parse.Error?  Is there another way to get the right type declared?   Still feels wrong to not extend FilesAdapter.   Seems to me they are actually dependent...

Reply:
OK.  Thought more about it.   I have an idea.

Reply:
All gone.